### ISSUE TITLE
Escaping variables does not work in Webpack dev server

### ISSUE THREAD

**Issue by mediaxtend** [lingui/js-lingui#745]
**Describe the bug**
When using React and the Webpack built-in development server (`$ yarn start`), the variables escaping does not work as expected.
The same test work as expected in bundled script (`$ yarn build`).
Variables escaping is done using a pair of ASCII apostrophes, as described in the [ICU messages format](http://userguide.icu-project.org/formatparse/messages#TOC-Quoting-Escaping): `This is an escaped variable: '{variable}'`.

**To Reproduce**
1. Create test project (using CRA):
```
$ yarn create react-app lingui-test
$ cd lingui-test
$ yarn add --dev @lingui/cli @lingui/macro @babel/core babel-core@bridge
$ yarn add @lingui/react
$ cat > .linguirc <<EOF
{
   "localeDir": "src/locales/",
   "srcPathDirs": ["src/"],
   "format": "po",
   "sourceLocale": "en"
}
EOF
```

2. Add `add-locale`, `extract` and `compile` commands to `scripts` section of  `package.json`.

3. Add locale (english only): `$ yarn add-locale en`

4. Modify `App.js`:

```
import React from 'react'; 
import { I18nProvider, withI18n } from '@lingui/react';
import { Trans, t } from '@lingui/macro';
import messagesEn from './locales/en/messages';

const Content = withI18n()(({ i18n }) => {
  const isoDate = (new Date()).toISOString();
  return (
    <div>
      <h1><Trans>Translated Title</Trans></h1>
      <p>Untranslated text with variable: {isoDate}</p>
      <p><Trans>Translated text with variable: {isoDate}</Trans> (using <code>Trans</code> tag macro)</p>
      <p>{i18n._(t`Translated text with variable: ${isoDate}`)} (using <code>t</code> function macro)</p>
      <p>{i18n._(t`Translated text with data placeholder: '{isoDate}'`).replace('{isoDate}', isoDate)} (using <code>t</code> function macro)</p>
    </div>
  );
});

const App = () => (
  <I18nProvider
    language="en"
    catalogs={{ en: messagesEn }}
    missing="🚨"
  >
    <Content />
  </I18nProvider>
);

export default App;
```

5. Extract messages and compile JS files: `$ yarn extract; yarn compile`

- Result using Webpack development server: start local server (`$ yarn start`) and open browser at http://localhost:3000/. I can see an error about getting property `isoDate` (`values` is `undefined`).

- Result using bundled script: build bundle (`$ yarn build`), start local web server (`$ serve -s build`) and open browser at http://localhost:5000/. I can see the page with the variable and the data placeholder correctly filled with the value.

In `messages.js`, messages are correctly parsed:
```
[…]
messages: {
    "Translated Title": "Translated Title",
    "Translated text with data placeholder: '{isoDate}'": "Translated text with data placeholder: {isoDate}",
    "Translated text with variable: {isoDate}": function(a) { return ["Translated text with variable: ", a("isoDate")] }
}
[…]
```

**Expected behavior**
The escaped variable should not be parsed. ``i18n._(t`This is an escaped variable: '{variable}'`)`` should return string `This is an escaped variable: {variable}`.

**Additional context**
Releases:
- jsLingui version `2.9.2`
- Babel version `7.11.6` and `7.9.0`
- Create React App (CRA) version `3.4.1`

**Comment by tricoder42** [comment]
Hey @mediaxtend, new version 3.x was released which includes lot of improvements. Please upgrade and check if the problem persists.

**Comment by mediaxtend** [comment]
It still does not work using Lingui 3.0.3 and CRA 4.0.
Using the Webpack development server, no error occurs but the placeholder has been cleared: `Translated text with data placeholder:`.
In built code, it's working correctly: `Translated text with data placeholder: 2020-11-01T…`.
Thanks in advance for your investigations.

**Comment by nickvirdenfdtrl** [comment]
Following @tricoder42's suggestion, upgrading from `2.9.1` to `3.3.0` resolved my issue with escaping variables in translations from my source language (English).

For anyone that ends up here like me looking to upgrade, this general config below works:

dependencies:
`@lingui/core`: `3.3.0`
`@lingui/macro`: `3.3.0`
`@lingui/react`: `3.3.0`
`react`: `16.13.0`

devDependencies:
`@babel/core`: `7.9.6`
`@lingui/cli`: `3.3.0`
`webpack`: `4.41.6`
`webpack-cli`: `3.3.0`
`webpack-dev-server`: `3.2.1`

And my `.linguirc` looks like:
```json
{
  "sourceLocale": "en",
  "rootDir": "./src/locales",
  "catalogs": [{
    "path": "<rootDir>/{locale}/messages",
    "include": ["<rootDir>"]
  }],
  "format": "po",
  "extractBabelOptions": {
    "plugins": ["@babel/plugin-syntax-dynamic-import"]
  },
  "locales": [
    "en",
    "es"
  ]
}
```

**Comment by semoal** [comment]
> Following @tricoder42's suggestion, upgrading from `2.9.1` to `3.3.0` resolved my issue with escaping variables in translations from my source language (English).
> 
> For anyone that ends up here like me looking to upgrade, this general config below works:
> 
> **dependencies**
> `@lingui/core`: `3.3.0`
> `@lingui/macro`: `3.3.0`
> `@lingui/react`: `3.3.0`
> `react`: `16.13.0`
> 
> **devDependencies**
> `@babel/core`: `7.9.6`
> `@lingui/cli`: `3.3.0`
> `webpack`: `4.41.6`
> `webpack-cli`: `3.3.0`
> `webpack-dev-server`: `3.2.1`
> 
> And my `.linguirc` looks like:
> 
> ```json
> {
>   "sourceLocale": "en",
>   "rootDir": "./src/locales",
>   "catalogs": [{
>     "path": "<rootDir>/{locale}/messages",
>     "include": ["<rootDir>"]
>   }],
>   "format": "po",
>   "extractBabelOptions": {
>     "plugins": ["@babel/plugin-syntax-dynamic-import"]
>   },
>   "locales": [
>     "en",
>     "es"
>   ]
> }
> ```

Yes this was resolved in 3.3.0, in 3.2.x not resolved yet. For future users :)
